closure/templates/test/ExampleModule.java

```diff
@@ -0,0 +1,81 @@
+package test;
```

Comment:
Please add a copyright header and reformat with `google-java-format`.

Reply:
Done